client/apps/shipping-label/view-wrapper-label.js

```diff
@@ -78,7 +79,12 @@ class ShippingLabelViewWrapper extends Component {
 
 		// eslint-disable-next-line no-undef
 		if ( wcConnectData.wcs_server_connection ) {
-			if ( 0 === events.length ) {
+
+			const labels = filter( events, { type: 'LABEL_PURCHASED' } );
+			const refunds = filter( events, { type: 'LABEL_REFUND_REQUESTED' } );
+			const activeLabels = differenceBy( labels, refunds, "labelIndex" );
```

Comment:
@budzanowski I am not 100% sure if `labelIndex` changes. Would `labelId` be better here?